cloud/awsprovider.go

```diff
@@ -1,10 +1,14 @@
 package cloud
 
 import (
+	"bytes"
```

Comment:
Be aware that by doing this, you have introduced a name collision with two or three variables below
![image](https://user-images.githubusercontent.com/22526/56441073-9ffda380-62a0-11e9-8d35-982bb35d4804.png)

Reply:
Switched to b. Thanks.